review section.js

```diff
@@ -0,0 +1,88 @@
+
+        paginatedReviews.forEach(review => {
+            const reviewElement = document.createElement('div');
+            reviewElement.classList.add('review');
+            reviewElement.innerHTML = `
+    <div class="review-content">
+        <div class="user">
+            <img src="${review['profile-pic']}" alt="${review.username}" class="profile-pic">
+            <h3 class="username">${review.username}</h3>
+            <p class="date-time">${new Date(review.date).toLocaleDateString()} ${new Date(review.date).toLocaleTimeString()}</p>
+        </div>
+            <div class="star-rating">${getStarRating(review.rating)}</div>
+            <p class="comment">${review.comment}</p>
+        <div class="like-dislike-buttons">
+            <div class="dislike-btn">
+                <button class="dislike-button">${review.dislikes} ${dislikeSvg} </button>
+            </div>
+            <div class="like-btn">
+                <button class="like-button">${review.likes} ${likeSvg} </button>
+            </div>
+        </div>
+    </div>
+`;
+            reviewsContainer.appendChild(reviewElement);
+        });
+    }
+
+
+    function setupPagination(reviews) {
+        paginationControls.innerHTML = '';
+        const totalPages = Math.ceil(reviews.length / reviewsPerPage);
+        for (let i = 1; i <= totalPages; i++) {
+            const button = document.createElement('button');
+            button.classList.add("pagination-button");
+            button.textContent = i;
+            button.addEventListener('click', () => {
+                currentPage = i;
+                displayReviews(reviews);
+            });
+            paginationControls.appendChild(button);
+        }
+    }
```

Comment:
poti folosit `scrollIntoView()` pe titlul `Reviews` (care are clasa `reviews`) pentru a scrolla inapoi sus cand schimbi pagina: https://stackoverflow.com/questions/71121040/how-to-use-scrollto-function-in-vanilla-js

Ce trebuie sa faci e sa salvezi intr-o constanta elementul cu clasa `reviews` si apoi pe el sa apelezi `.scrollIntoView()`


https://github.com/user-attachments/assets/b8cc71d7-fcf8-4ab5-bf66-87ad23f368a3